packages/desktop-client/src/components/Settings.js

```diff
@@ -473,6 +473,42 @@ function SettingsLink({ to, name, style, first, last }) {
   );
 }
 
+function Version() {
+  let [version, setVersion] = useState('');
+
+  useEffect(async () => {
+    const url = await send('get-server-url');
+    if (!url || url.indexOf('not-configured') !== -1) return;
+
+    try {
+      const res = await fetch(url + '/info');
```

Comment:
Generally, the frontend shouldn't ever hit the server directly. It should hit a backend method that talks to the server. Just makes things less confusing when it comes to mixing non-auth and auth methods on the server, etc. You couldn't do this if this endpoint needed to be authenticated, and it's nice to keep the workflow consistent

I'm not 100% sure about making this `/info` endpoint something the app relies but, but I guess it's OK for now. It's provided by a node-specific library and we haven't really thought through the shape of the API. We can probably change it later though.

Can you instead hit it via the backend, and expose a new backend method (it can be called `get-server-version` and live beside `get-server-url`)?